data/allPossibleCategories.ts

```diff
@@ -0,0 +1,37 @@
+module.exports = function () {
+	// const allPossibleCategoriesRaw = require('@joplin/lib/pluginCategories.json');
+
+	// interface Category {
+	// 	name: string
+	// }
+
+	// allPossibleCategories will be obtained from @joplin/lib in the next step
+	const allPossibleCategoriesRaw = [
+		'Appearance',
+		'Developer Tools',
+		'Productivity',
+		'Themes',
+		'Integrations',
+		'Viewer',
+		'Search',
+		'Tags',
+		'Editor',
+		'Files',
+		'Personal Knowledge Management',
+	]
+
+
+	const allPossibleCategories = allPossibleCategoriesRaw.map((category) => {
+		return {
+			name: category.toLowerCase().replace(/[ ]/g, '-'),
+			displayName: category,
+		}
+	})
+	// const allPossibleCategories = allPossibleCategoriesRaw.map((category: Category) => {
+	// 	return {
+	// 		name: category.name.replace(/[ ]/g, '-'),
+	// 		displayName: category,
+	// 	}
+	// })
```

Comment:
Consider removing this commented-out code (or adding a comment to explain why it's commented out).

Reply:
these codes are prepared for pulling this list from joplin/lib. Maybe just leave them here for now, I'll replace them when lib v2.9 is released.